palindrome_algorithm/test_palindrome.js

```diff
@@ -0,0 +1,7 @@
+const palindrome = require('./palindrome');
+
+console.assert(palindrome("racecar"));
+console.assert(!palindrome("holiday"));
+console.assert(palindrome("dad"));
+console.assert(!palindrome("sarcasm"));
+console.assert(palindrome("mom"));
```

Comment:
[**Blocker** (PR comments with a purpose)](https://gist.github.com/echolabstech/a905ad02efba1ef6663c96ee4f251480):
please do blah blah, this is a bug